packages/cli/src/commands/link/android/patches/makeBuildPatch.js

```diff
@@ -7,17 +7,40 @@
  * @format
  */
 
+import fs from 'fs';
 import normalizeProjectName from './normalizeProjectName';
 
-export default function makeBuildPatch(name) {
+export default function makeBuildPatch(name, buildGradlePath) {
   const normalizedProjectName = normalizeProjectName(name);
   const installPattern = new RegExp(
-    `(implementation|api|compile)\\w*\\s*\\(*project\\(['"]:${normalizedProjectName}['"]\\)`,
+    `(implementation|api|compile)\\w*\\s*\\(*project\\s*\\(['"]:${normalizedProjectName}['"]\\)`,
```

Comment:
How about we extract https://github.com/react-native-community/react-native-cli/pull/221/files#diff-7397ff56d9926b7ce487df7df7efbf8dR37 to a separate function and make `installPattern` based on it? They seem to be the same and I am afraid of having them defined in two places.

Reply:
Right, better to have a single source of truth. The same goes for the dependency types.